muckrock/jurisdiction/api_v2/serializers.py

```diff
@@ -74,3 +74,26 @@ class Meta:
             "level",
             "parent",
         )
+        extra_kwargs = {
```

Comment:
`jurisdiction/viewsets.py` isn't in this diff, but it needs help text on its filters - abbrev, level, name, parent